meqa/mqgen/main.go

```diff
@@ -32,50 +32,23 @@ func main() {
 	swaggerFile := flag.String("s", swaggerJSONFile, "the swagger.yml file location")
 	algorithm := flag.String("a", "all", "the algorithm - simple, object, path, all")
 	verbose := flag.Bool("v", false, "turn on verbose mode")
-	whitelistFile := flag.String("w", "", "the whitelisted APIs file location")
+	whitelistedAPIsFile := flag.String("w", "", "the whitelisted APIs file location")
 	ignoredPathsFile := flag.String("m", "", "the paths in this file will be ignored")
```

Comment:
Please provide name of the file (that lists out all ignored paths in APIs) along with its relative path. Example testdata/ignorePaths.cfg